blocks/embed/embed.js

```diff
@@ -76,20 +94,29 @@ const loadEmbed = (block, link, autoplay) => {
   ];
 
   const config = EMBEDS_CONFIG.find((e) => e.match.some((match) => link.includes(match)));
-  const url = new URL(link);
+  const url = link ? new URL(link) : '';
+  const blockConfig = readBlockConfig(block);
+
   if (config) {
     block.innerHTML = config.embed(url, autoplay);
     block.classList = `block embed embed-${config.match[0]}`;
+  } else if (block.classList.contains('brightcove')) {
+    const video = blockConfig.video ? blockConfig.video : '6331538546112';
+    const account = blockConfig.account ? blockConfig.account : '5703385908001';
+    const player = blockConfig.player ? blockConfig.player : 'default';
```

Comment:
JS syntax for this is `const account = blockConfig.account || '5703385908001';`

Ternary operator obviously works, but wanted to share.

Reply:
Ok, it's less verbose your way. I will refactor.